### DISCUSSION TITLE
Zwei HA Instanzen: eine davon mit RaspberryMatic Addon, Zugriff zweite Instanz auf RaspberryMatic der ersten Instanz einrichten

### DISCUSSION THREAD

**Discussion by Andre-Acer** [SukramJ/aiohomematic#1726] [Q&A]
Hallo!

Wir sind zwei Haushalte, teilen uns aber ein Netzwerk. Wir haben daher zwei getrennte Home Assistant Instanzen in einem Netzwerk - HA Instanz Haushalt 1 und HA Instanz Haushalt 2. 

In Haushalt 1 ist RaspberryMatic als Addon unter HA installiert. Der Zugriff über Homematic(IP) Local klappt dort problemlos. 
In Haushalt 2 möchte wir nun über Homematic(IP) local Zugriff auf die RaspberryMatic in Haushalt 1 bekommen. Wie gesagt: beide Instanzen im selben Netzwerk, keine getrennten Subnetze oder dergleichen.

Für das RaspberryMatic Addon in Haushalt 1 habe ich bereits diverse Portfreigaben im Addon gemacht:

![image](https://github.com/user-attachments/assets/2b19b0b9-1288-481a-a4e9-1bd74e3859e9)

Wenn ich nun im Homematic(IP) Local in Haushalt 2 versuche die RaspberryMatic aus Haushalt 1 zu verbinden, dann erhalte ich nicht mal eine Verbindung:

![image](https://github.com/user-attachments/assets/ff1a97db-076a-489a-8701-df5ccff57533)

Fehlen mir noch Port-Freigaben und ich müsste über einen anderen Port kommen?

**Comment by SukramJ** [comment]
Ip-Adressen werden ohne Ports eingetragen.
Die 8080 ist der JSON-Port.
